tests/e2e/tests/checkout/checkout-block-place-order.block_theme.side_effects.spec.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { test as base, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import { CheckoutPage } from './checkout.page';
+import {
+	SIMPLE_PHYSICAL_PRODUCT_NAME,
+	FREE_SHIPPING_NAME,
+	FREE_SHIPPING_PRICE,
+} from './constants';
+
+const test = base.extend< { pageObject: CheckoutPage } >( {
+	pageObject: async ( { page }, use ) => {
+		const pageObject = new CheckoutPage( {
+			page,
+		} );
+		await use( pageObject );
+	},
+} );
+
+test.describe( 'Shopper → Checkout block → Place Order → customer', () => {
+	test.use( { storageState: 'playwright/.auth/customer.json' } );
+
+	test( 'Guest user can place order', async ( {
```

Comment:
Think you got the name of this test mixed up with the other, this is for logged in users, not guests (based on the storage state we're loading)